opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/metric_reader_storage.py

```diff
@@ -152,16 +152,21 @@ def collect(self) -> MetricsData:
 
                 for view_instrument_match in view_instrument_matches:
 
+                    data_points = view_instrument_match.collect(
+                        aggregation_temporality, collection_start_nanos
+                    )
+
+                    if data_points is None:
+                        continue
```

Comment:
See comment above, but we could remove the `None` case and just check if its empty here